R/extract_soil_map_data.R

```diff
@@ -0,0 +1,62 @@
+#' @title Extract soil properties from Flemish soil map
+#'
+#' @description This function queries the Flemish soil map attributres at a given coordinate,
+#' http://www.geopunt.be/catalogus/datasetfolder/5c129f2d-4498-4bc3-8860-01cb2d513f8f
+#' by using the affiliated WFS service provided by DOV. The user can pick the
+#' properties of interest. A full list of properties is available at
+#' https://www.dov.vlaanderen.be/geoserver/bodemkaart/bodemtypes/wfs?request=DescribeFeatureType
+#' Coordinates should be given as Lambert 72 (epsg 31370). When outside the
```

Comment:
See https://epsg.org/search/by-name?sessionkey=lqlp8f7hg1&searchedTerms=lambert+72 : the name is best put more specifically.

```suggestion
#' Coordinates should be given in the 'Belge 1972 / Belgian Lambert 72' 
#' (\href{https://epsg.io/31370}{EPSG:31370}) coordinate reference
#' system. When outside the
```